### PR TITLE
Forward-port italian translation changes from release/0.81.x

### PR DESCRIPTION
**Pull request by farsil**
# Description

Forward port italian translation changes to main from `release/0.81.x`. Better getting this done now than later :)

# Manual testing

N/A

# Checklist

_Please tick the items as you have addressed them. Don't remove items; leave the ones that are not applicable unchecked._

I have:

- [x] followed the project's [contributing guidelines](https://github.com/dosbox-staging/dosbox-staging/blob/master/CONTRIBUTING.md) and [code of conduct](https://github.com/dosbox-staging/dosbox-staging/blob/master/CODE_OF_CONDUCT.md).
- [x] performed a self-review of my code.
- [ ] commented on the particularly hard-to-understand areas of my code.
- [ ] split my work into well-defined, bisectable commits, and I [named my commits well](https://github.com/dosbox-staging/dosbox-staging/blob/main/CONTRIBUTING.md#commit-messages).
- [ ] applied the appropriate labels (bug, enhancement, refactoring, documentation, etc.)
- [ ] [checked](https://github.com/dosbox-staging/dosbox-staging/blob/main/scripts/compile_commits.sh) that all my commits can be built.
- [ ] confirmed that my code does not cause performance regressions (e.g., by running the Quake benchmark).
- [ ] added unit tests where applicable to prove the correctness of my code and to avoid future regressions.
- [ ] made corresponding changes to the documentation or the website according to the [documentation guidelines](https://github.com/dosbox-staging/dosbox-staging/blob/main/DOCUMENTATION.md).
- [ ] [locally verified](https://github.com/dosbox-staging/dosbox-staging/blob/main/DOCUMENTATION.md#previewing-documentation-changes-locally) my website or documentation changes.

### CODE REVIEW

File: contrib/resources/translations/it.lng [hunk 1]
```diff
@@ -617,55 +614,6 @@ Usa 'allow' (consenti) o 'block' (blocca) per sovrascrivere la variabile
...
-                      la visualizzazione dell'impostazione dei cicli.
-  mouse=<valore>:     Livello di verbosità dei messaggi di cattura del mouse:
-                        none/off:  Non visualizza alcun messaggio.
-                        short:     Visualizza solo il messaggio di cattura
-                                   del mouse.
-                        full:      Visualizza informazioni dettagliate su come
-                                   catturare o rilasciare il cursore
-                                   (predefinito).
-.
 :CONFIG_LANGUAGE
```
Comment (johnnovak) [inline contrib/resources/translations/it.lng:617]: Are you sure you wanted to delete this?

File: contrib/resources/translations/it.lng [hunk 2]
```diff
@@ -1468,13 +1431,6 @@ scrittura della porta I/O:
...
                                   su un periodo di 3s.)
 .
-:CONFIG_OPL_REMOVE_DC_BIAS
-Rimuove la polarizzazione CC dall'uscita OPL. Dovrebbe essere usato solo
-come ultima risorsa per correggere gli scoppiettii nel suono nei giochi che
-riproducono audio PCM utilizzando il sintetizzatore OPL su una scheda
-Sound Blaster o AdLib, come in: Golden Eagle (1991), Wizardry 6 (1990) e
-Wizardry 7 (1992). Si prega di segnalare altri giochi interessati.
-.
 :CONFIG_OPLEMU
```
Comment (johnnovak) [inline contrib/resources/translations/it.lng:1434]: Are you sure you wanted to delete this?

File: contrib/resources/translations/it.lng [hunk 3]
```diff
@@ -2112,9 +2068,6 @@ File eseguibile non trovato: '%s'
 :CONJUNCTION_AND
 e
 .
-:PROGRAM_CONFIG_NOT_CHANGEABLE
-La proprietà '%s' non è modificabile durante l'esecuzione.
-.
 :MIDI_DEVICE_LIST_NOT_SUPPORTED
```
Comment (johnnovak) [inline contrib/resources/translations/it.lng:2071]: Are you sure you wanted to delete this?

File: contrib/resources/translations/it.lng [hunk 4]
```diff
@@ -449,9 +449,6 @@ MUTO
 :TITLEBAR_PAUSED
 PAUSA
 .
-:TITLEBAR_HINT_CAPTURED
-mouse catturato
-.
 :TITLEBAR_HINT_CAPTURED_HOTKEY
```
Comment (johnnovak) [inline contrib/resources/translations/it.lng:452]: Are you sure you wanted to delete this?

File: contrib/resources/translations/it.lng (hunk 1 above)
Comment (farsil) [reply contrib/resources/translations/it.lng:617]: No, that's why I put it back in draft 😁 I think I made a mess with rebasing

Comment (johnnovak) [reply contrib/resources/translations/it.lng:617]: Sure, no rush :)

**Comment by johnnovak** [comment]
Closing due to inactivity after 3 weeks. Please open a new PR @farsil when you're ready.

**Pull request closed without merging**